sdk/azidentity/chained_token_credential.go

```diff
@@ -30,7 +30,7 @@ func NewChainedTokenCredential(sources ...azcore.TokenCredential) (*ChainedToken
 	return &ChainedTokenCredential{sources: sources}, nil
 }
 
-// GetToken sequentially calls TokenCredential.GetToken on all the specified sources, returning the first non default AccessToken.
+// GetToken sequentially calls TokenCredential.GetToken on all the specified sources, returning the first non-nil AccessToken.
```

Comment:
`returning the token from the first successful call to GetToken()`

Reply:
Done